src/net.cpp

```diff
@@ -1841,11 +1841,11 @@ void CConnman::ThreadOpenConnections(const std::vector<std::string> connect)
                 // but inbound and manual peers do not use our outbound slots. Inbound peers
                 // also have the added issue that they could be attacker controlled and used
                 // to prevent us from connecting to particular hosts if we used them here.
-                switch(pnode->m_conn_type){
+                switch (pnode->m_conn_type){
```

Comment:
style-nit:

```suggestion
                switch (pnode->m_conn_type) {
```

if you decide to do style-fixups, it would probably be good to do all of them in one go or not at all. Maybe with clang-format-diff?   https://github.com/bitcoin/bitcoin/tree/master/contrib/devtools#clang-format-diffpy

Reply:
heh. thanks for the nudge. I've updated my tooling so I'll pay better attention to the clang-format-diff moving forward. I think I've now cleaned everything up properly in this PR? I'm not sure what exactly you mean by "all in one go or not at all"... 

(disclaimer: I didn't take the suggestions to not [IndentCaseBlocks](https://clang.llvm.org/docs/ClangFormatStyleOptions.html). I would much prefer if we turned that format option to true.)